src/blocks/weather.rs

```diff
@@ -103,7 +112,7 @@ fn default_interval() -> Seconds {
 
 #[derive(Deserialize, Debug)]
 #[serde(tag = "name", rename_all = "lowercase")]
-enum WeatherService {
+pub enum WeatherService {
     OpenWeatherMap {
```

Comment:
You can avoid duplicating code by doing this
```rust
#[derive(Deserialize, Debug)]
#[serde(tag = "name", rename_all = "lowercase")]
pub enum WeatherService {
    OpenWeatherMap(open_weather_map::OpenWeatherMapConfig),
    MetNo(met_no::Config),
}
```

Reply:
Ah, of course! I saw that this was a bad pattern, but couldn't free myself from the struct enums.